### PR TITLE
docs: Upgrade readme

### PR DESCRIPTION
**Pull request by tlgimenes**
## What's the purpose of this pull request?
README info was quite old already. This PR updates this info and add links to our wonderful docs. Next step is to add content to our docs.

## How it works? 
I was inspired into Gatsby and Next.JS README. The idea is to give a glimpse of the project in the README and add links to the main docs where the developer can actually read content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,134 +1,38 @@
...
+    Fullstack ecommerce toolkit
+  </strong>
+</p>
+<p align="center">
+  FastStore is a fullstack toolkit based on React that helps developers build Performant, Stable, SEO and Analytics ready ecommerces in a matter of days.
+</p>
+
+- <strong>Performance</strong>: Keep yourself calm. We use strict budgets to ensure our libs are always light and fast.
+- <strong>Stability</strong>: Generate your pages ahead by using a Jamstack ready toolkit.
+- <strong>SEO/Analytcs</strong>: Connect to your favorite analytics provider
```
Comment (lariciamota) [inline README.md:20]:
```suggestion
- <strong>SEO/Analytics</strong>: Connect to your favorite analytics provider.
```

File: README.md [hunk 2]
```diff
@@ -1,134 +1,38 @@
...
+- <strong>Freedom</strong>: Use our API/SDK/UI components to connect, manage and display your ecommerce in your favorite React framework 
+(Gatsby, Next.JS, Remix etc)
+- <strong>Speed</strong>: Customize one of our starters to your needs. Do not spend time creating tests, thinkig about architecture or infrastructure.
```
Comment (lariciamota) [inline README.md:23]:
```suggestion
- <strong>Speed</strong>: Customize one of our starters to your needs. Do not waste time creating tests, thinking about architecture or infrastructure, it's all taken care of.
```

File: README.md [hunk 3]
```diff
@@ -1,134 +1,38 @@
...
+
+## Getting started
+[Checkout the docs](https://faststore.dev/quickstart) to quickstart from one of our official starters or create your own.
```
Comment (lariciamota) [inline README.md:26]:
```suggestion
[Check out the docs](https://faststore.dev/quickstart) to quickstart from one of our official starters or create your own.
```

File: README.md [hunk 4]
```diff
@@ -1,134 +1,38 @@
...
+
+## Official Starters
+- [Base](https://github.com/vtex-sites/base.store): Basic starter with generic store.
+
+## Learn
+Check out our [Tutorials](https://faststore.dev/tutorials/overview), [How-to Guides](https://faststore.dev/how-to-guides/overview), [API References](https://faststore.dev/reference/overview) and [Conceptual guides](https://faststore.dev/conceptual-guides/overview) to deep dive in the world of building FastStores
```
Comment (lariciamota) [inline README.md:32]:
```suggestion
Check out our [Tutorials](https://faststore.dev/tutorials/overview), [How-to Guides](https://faststore.dev/how-to-guides/overview), [API References](https://faststore.dev/reference/overview) and [Conceptual guides](https://faststore.dev/conceptual-guides/overview) to deep dive in the world of building FastStores.
```

File: README.md [hunk 5]
```diff
@@ -1,134 +1,38 @@
...
+
+## Release Notes
+Follow our montly release notes at: https://faststore.dev/releases
```
Comment (lariciamota) [inline README.md:35]:
```suggestion
Follow our monthly release notes at: https://faststore.dev/releases
```

File: README.md [hunk 6]
```diff
@@ -137,21 +41,20 @@ These are the clients running in production with FastStore:
...
-
-**Answer:** You need to log in VTEX by `vtex login <account>`
+## Contributing
+This is still a work in progress, however, if you are also an adventurous person, you can read the code and have some fun!
 
----
+## Getting help
 
-**Question:** I'm trying to add a dependency inside a specific package with `yarn add <package>`, but it's not working.
+If you find any issues on the project you would like to report, please create an [issue](https://github.com/vtex/faststore/issues) on the repository. If you have a question, idea or want to show us something cool you have built, feel free to create a [discussion](https://github.com/vtex/faststore/discussions).
```
Comment (lariciamota) [inline README.md:49]:
```suggestion
If you find any issues on the project you would like to report, please create an [issue](https://github.com/vtex/faststore/issues) on the repository. If you have a question, idea, or want to show us something cool you have built, feel free to create a [discussion](https://github.com/vtex/faststore/discussions).
```

File: README.md [hunk 7]
```diff
@@ -137,21 +41,20 @@ These are the clients running in production with FastStore:
...
+| :--------------------------- | :---------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------: |
+| @faststore/ui               | Exports basic Store components. Checkout our [storybook](https://faststoreui.netlify.app/)                                                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui)  [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![three shakig](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![three shakig](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)            |
```
Comment (lariciamota) [inline README.md:55]:
```suggestion
| @faststore/ui               | Exports basic Store components. Check out our [storybook](https://faststoreui.netlify.app/)                                                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui)  [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![three shakig](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![three shakig](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)            |
```

**Review by lariciamota — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -137,21 +41,20 @@ These are the clients running in production with FastStore:
...
+| :--------------------------- | :---------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------: |
+| @faststore/ui               | Exports basic Store components. Check out our [storybook](https://faststoreui.netlify.app/)                                                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui)  [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![three shakig](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![three shakig](https://badgen.net/bundlephobia/dependency-count/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)            |
```
Comment (filipewl) [inline README.md:55]: Noticed this duplicate badge for tree shaking:

```suggestion
| @faststore/ui               | Exports basic Store components. Check out our [storybook](https://faststoreui.netlify.app/)                                                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fui.svg)](https://badge.fury.io/js/%40faststore%2Fui)  [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/ui)](https://bundlephobia.com/package/@faststore/ui)            |
```

File: README.md [hunk 9]
```diff
@@ -137,21 +41,20 @@ These are the clients running in production with FastStore:
...
+| @faststore/sdk               | Exports basic logic hooks for creating your custom ecommerce                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fsdk.svg)](https://badge.fury.io/js/%40faststore%2Fsdk) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![three shakig](https://badgen.net/bundlephobia/dependency-count/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk)  
+| @faststore/api               | Connect your site to your ecommerce provider                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fapi.svg)](https://badge.fury.io/js/%40faststore%2Fapi) [![npm version](https://badgen.net/bundlephobia/dependency-count/@faststore/api)](https://bundlephobia.com/package/@faststore/api)
```
Comment (filipewl) [inline README.md:57]: Also a duplicate NPM version badge:

```suggestion
| @faststore/api               | Connect your site to your ecommerce provider                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fapi.svg)](https://bundlephobia.com/package/@faststore/api)     
```

File: README.md [hunk 10]
```diff
@@ -137,21 +41,20 @@ These are the clients running in production with FastStore:
...
+| @faststore/sdk               | Exports basic logic hooks for creating your custom ecommerce                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fsdk.svg)](https://badge.fury.io/js/%40faststore%2Fsdk) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![three shakig](https://badgen.net/bundlephobia/dependency-count/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk)
```
Comment (filipewl) [inline README.md:56]: Minor typo:

```suggestion
| @faststore/sdk               | Exports basic logic hooks for creating your custom ecommerce                      |               [![npm version](https://badge.fury.io/js/%40faststore%2Fsdk.svg)](https://badge.fury.io/js/%40faststore%2Fsdk) [![size minzip](https://badgen.net/bundlephobia/minzip/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/tree-shaking/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk) [![tree shaking](https://badgen.net/bundlephobia/dependency-count/@faststore/sdk)](https://bundlephobia.com/package/@faststore/sdk)  
```

**Review by filipewl — APPROVED** [review]
LGTM!

Tried figuring out why some badges weren't showing up but couldn't figure out. It's probably something temporary on their end.
Screen Shot 2021-12-08 at 17 53 17

**Review by carolinamenezes — APPROVED** [review]

**Pull request merged**